commands/alpha/rpkg/clone/command.go

```diff
@@ -200,6 +204,24 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func (r *runner) packageAlreadyExists(packageName string) error {
+	// only the first package revision can be created from init or clone, so
+	// we need to check that the package doesn't already exist.
+	packageRevisionList := porchapi.PackageRevisionList{}
+	if err := r.client.List(r.ctx, &packageRevisionList, &client.ListOptions{
+		Namespace: *r.cfg.Namespace,
+	}); err != nil {
+		return err
+	}
+	for _, pr := range packageRevisionList.Items {
+		if pr.Spec.RepositoryName == r.repository && pr.Spec.PackageName == packageName {
```

Comment:
I think we support fieldSelectors today, but if/when we get to CRDs we likely won't, so ... fine for now!